game_launch/game.cpp

```diff
@@ -138,7 +139,12 @@ _inline int Sys_Start( void )
 		changeGame = Sys_ChangeGame;
 #endif
 
-	ret = Xash_Main( szArgc, szArgv, GAME_PATH, 0, changeGame );
+	char *game = getenv( E_GAME  );
```

Comment:
make it `const char` please, so we won't lose const modifier two lines below. Otherwise lgtm